x-pack/plugins/serverless_search/public/application/components/overview.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+  EuiPageTemplate,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useState } from 'react';
+import { PLUGIN_ID } from '../../../common';
+import { useKibanaServices } from '../hooks/use_kibana';
+import { CodeBox } from './code_box';
+import { javascriptDefinition } from './languages/javascript';
+import { languageDefinitions } from './languages/languages';
+import { LanguageDefinition } from './languages/types';
+import { InstallClientPanel } from './overview_panels/install_client';
+import { OverviewPanel } from './overview_panels/overview_panel';
+import './overview.scss';
+import { IngestData } from './overview_panels/ingest_data';
+
+export const ElasticsearchOverview = () => {
+  const [selectedLanguage, setSelectedLanguage] =
+    useState<LanguageDefinition>(javascriptDefinition);
```

Comment:
:+1: this is good for now. 

We may want to consider saving this to local storage and initializing from there so it doesn't always change back. That could also be useful later as we add more pages.